app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store references to `Person` instances by their name. Ensure that names are unique to avoid overwriting entries in this dictionary.